legal/Container-Images-Legal-Notice.md

```diff
@@ -1 +1,3 @@
-**Legal Notice:** Microsoft containers are licensed under various licenses depending upon the container’s content. If the container has a Debian, Ubuntu or Alpine Linux base image and you want to retrieve the licenses, copyright notices or source code information for the base image and its official packages, please review the [Linux Legal Metadata](https://aka.ms/mcr/osslinuxmetadata) instructions file. If you are using a Windows based container the governing licenses are referenced in this Docker Hub “License” section or in the container itself, usually as a NOTICE or LICENSE file.  All other components (whether open or closed source) are licensed as indicated either in this Docker Hub “License” section or in the container itself.
\ No newline at end of file
+**Legal Notice:** Microsoft containers are licensed under various licenses depending upon the container’s content. If the container has a Debian, Ubuntu or Alpine Linux base image and you want to retrieve the licenses, copyright notices or source code information for the base image and its official packages, please review the [Linux Legal Metadata](https://aka.ms/mcr/osslinuxmetadata) instructions file. If you are using a Windows based container the governing licenses are referenced in this Docker Hub “License” section or in the container itself, usually as a NOTICE or LICENSE file.  All other components (whether open or closed source) are licensed as indicated either in this Docker Hub “License” section or in the container itself.
+
+This container may incorporate material from third parties. Microsoft makes certain open source code available at [https://3rdpartysource.microsoft.com](https://3rdpartysource.microsoft.com), or if you want a copy of the source code in this container you may send a check or money order for US $5.00, including the product name, the open source component name, and version number, to: Source Code Compliance Team, Microsoft Corporation, One Microsoft Way, Redmond, WA 98052 USA
```

Comment:
The `This container` wording is a little strange to me given the context.  `Microsoft containers` or `containers` is more understandable IMO.

```suggestion
Microsoft containers may incorporate material from third parties. Microsoft makes certain open source code available at [https://3rdpartysource.microsoft.com](https://3rdpartysource.microsoft.com), or if you want a copy of the source code in this container you may send a check or money order for US $5.00, including the product name, the open source component name, and version number, to: Source Code Compliance Team, Microsoft Corporation, One Microsoft Way, Redmond, WA 98052 USA
```